information-system/data-store/access-control.md

```diff
@@ -0,0 +1,101 @@
+- If disabled, general users will not be able to update the metadata.
+
+## Data access
+
+Facilitates control over the upload/download of data to/from the storage location associated with the dataset.
+
+### Data read
+
+- If enabled, the download panel will become visible to users and they will be able to download data from the storage location. 
+- If disabled, the download panel will not be visible and users will not be able to download data from the storage location.
+
+### Data write
+
+- If enabled, users will be able to upload/overwrite and download to/from the storage location as both the upload and download panels will be made available to general users.
+- If disabled, users will not be able to upload/overwrite data in the storage location.
+
+## Admin access
+
+- If enabled, provides administrative control over the dataset. That is, the ability to lock and unlock the dataset in addition to all of the above privileges. For more information on locking and unlocking, see here.
+
+# Applying Access Controls
+
+The aforementioned controls can be applied for individual datasets by those who have administration access to the dataset. By default, this is the owner of the dataset and Information System Admins.
+
+Each dataset can have a rule for its accessibility to (the group of all) general datastore users, and (optionally) extra rules for multiple custom groups of users. The following sections will discuss these.  
+
+## General user access to dataset
+
+### Who are general users?
+
+General Datastore users refers to the group of all users who can access the datastore. As such, general user access controls facilitate the general privacy or publicity (to all datastore users) of a dataset. You could limit access controls for general users if you wish to keep your dataset private to a group (perhaps during development processes and then later release it by updating general user access rights).
+
+### Where to manage general user access
+
+Upon registering a dataset, the general datastore user access rights can be managed by navigating to the 'settings' tab when viewing the dataset.
+
+In future versions of the IS, the ability to set access levels during the registration process will be implemented. For now, a workaround is to register a dataset with meaningless metadata, limit general access privileges, then meaningfully update the metadata as you wish.
+
+## Group access to dataset
+
+The group access feature enables custom permissions for groups (and thus members) to a dataset. This can achieve the privatisation of datasets to only certain users of the information system by *not* enabling general user read/write permissions and creating a group with enabled read/write permissions for its members.
+
+### Creating a group
+
+Group creation is currently performed by Information System Administrators. Please get in touch and we can help facilitate this for you.
```

Comment:
Jonathan to add section here.

Reply:
Out of scope for this PR - still okay to approve assuming other stuff resolved

Reply:
@jyucsiro  just noting that this will require your input here regarding the proper procedure to request a group.